sdk/communication/Azure.Communication.Chat/tests/SessionRecords/ChatClientsLiveTests/ReadReceiptGS.json

```diff
@@ -1,55 +1,49 @@
 {
```

Comment:
To avoid unnecessary conflicts, it is probably better to change your commit history and not include updating the tests that are not affected by your change

Reply:
I will abandon this PR and create a new one as I've rebased and the new iteration could show lot of unnecessary changes that got pulled in.